### PR TITLE
[EIAnalytics]Replace searchbox with material ui autocomplete

### PR DESCRIPTION
**Pull request by ruwiniwj**
## Purpose
The EIAnalytics searchbox was previously written using only react select. Its now replaced with material ui autocomplete react select.

### CODE REVIEW

File: components/ei-analytics/org.wso2.analytics.solutions.ei.analytics/widgets/EIAnalyticsSearchBox/src/EIAnalyticsSearchBox.jsx [hunk 1]
```diff
@@ -16,24 +16,205 @@
...
+
+const lightTheme = createMuiTheme({
+    palette: {
+        type: 'light'
+    }
+});
+
+let openPopper = false;
+
+function NoOptionsMessage(props) {
```
Comment (this) [inline components/ei-analytics/org.wso2.analytics.solutions.ei.analytics/widgets/EIAnalyticsSearchBox/src/EIAnalyticsSearchBox.jsx]: Why these functions resides outside a class?

Comment (ruwiniwj) [reply components/ei-analytics/org.wso2.analytics.solutions.ei.analytics/widgets/EIAnalyticsSearchBox/src/EIAnalyticsSearchBox.jsx]: That's how material ui implements autocomplete using react select.

File: components/ei-analytics/org.wso2.analytics.solutions.ei.analytics/widgets/EIAnalyticsSearchBox/src/EIAnalyticsSearchBox.jsx [hunk 2]
```diff
@@ -141,39 +314,173 @@ class EIAnalyticsSearchBox extends Widget {
...
+                            style={{
+                                paddingLeft: 15,
+                                paddingRight: 15,
+                                paddingTop: 5,
+                            }}>
+                            <div>
+                                Filter by {this.formatPageName(this.pageName)}
+                            </div>
+                            <div>
+                                <Select
```
Comment (this) [inline components/ei-analytics/org.wso2.analytics.solutions.ei.analytics/widgets/EIAnalyticsSearchBox/src/EIAnalyticsSearchBox.jsx]: @ruwiniwj Please add a TODO to move this `<Select>` component and related stuff to a separate component as this class have become very long.

File: components/ei-analytics/org.wso2.analytics.solutions.ei.analytics/widgets/EIAnalyticsSearchBox/src/EIAnalyticsSearchBox.jsx [hunk 3]
```diff
@@ -109,19 +264,37 @@ class EIAnalyticsSearchBox extends Widget {
 
         // remove sequences in the excludeSequences-array from the options
         else if (this.pageName == this.pgSequence) {
-            let excludeSequences = ["PROXY_INSEQ", "PROXY_OUTSEQ", "PROXY_FAULTSEQ", "API_OUTSEQ", "API_INSEQ", "API_FAULTSEQ", "AnonymousSequence"];
+            let excludeSequences = ["PROXY_INSEQ", "PROXY_OUTSEQ", "PROXY_FAULTSEQ", "API_OUTSEQ", "API_INSEQ",
```
Comment (TanyaM) [inline components/ei-analytics/org.wso2.analytics.solutions.ei.analytics/widgets/EIAnalyticsSearchBox/src/EIAnalyticsSearchBox.jsx]: Let's stick to single quotes

File: components/ei-analytics/org.wso2.analytics.solutions.ei.analytics/widgets/EIAnalyticsSearchBox/src/EIAnalyticsSearchBox.jsx (hunk 2 above)
Comment (ruwiniwj) [reply components/ei-analytics/org.wso2.analytics.solutions.ei.analytics/widgets/EIAnalyticsSearchBox/src/EIAnalyticsSearchBox.jsx]: Fixed with https://github.com/wso2/analytics-solutions/pull/114

File: components/ei-analytics/org.wso2.analytics.solutions.ei.analytics/widgets/EIAnalyticsSearchBox/src/EIAnalyticsSearchBox.jsx (hunk 3 above)
Comment (ruwiniwj) [reply components/ei-analytics/org.wso2.analytics.solutions.ei.analytics/widgets/EIAnalyticsSearchBox/src/EIAnalyticsSearchBox.jsx]: Fixed with #303

**Pull request merged**